betty/extension/cotton_candy/assets/templates/notes.html.j2

```diff
@@ -0,0 +1,14 @@
+{% set notes = notes | select('public') | list %}
```

Comment:
Rename this section-notes.html.j2 and add aliases for similar templates as well. Or do that in a follow-up.